env/dev/env/ios/main.cljs

```diff
@@ -4,19 +4,26 @@
             [status-im.ios.core :as core]
             [status-im.utils.handlers :as utils.handlers]
             [figwheel.client :as figwheel :include-macros true]
+            [env.config :as conf]
+            [env.utils]
             [cljs.pprint]))
 
 (enable-console-print!)
 
+(assert (exists? core/init) "Fatal Error - Your core.cljs file doesn't define an 'init' function!!! - Perhaps there was a compilation failure?")
+(assert (exists? core/app-root) "Fatal Error - Your core.cljs file doesn't define an 'app-root' function!!! - Perhaps there was a compilation failure?")
+
 (def cnt (r/atom 0))
 (defn reloader [] @cnt [core/app-root])
 (def root-el (r/as-element [reloader]))
 
-(figwheel/watch-and-reload
-  :websocket-url "ws://localhost:3449/figwheel-ws"
-  :heads-up-display false
-  :jsload-callback #(swap! cnt inc))
+(figwheel/start {
```

Comment:
don't add linebreak after opening brackets

Reply:
@yenda Thanks for your feedback. I will address theses issues late next week.